docs/extend/modular-resources/configure.md

```diff
@@ -193,46 +194,89 @@ The custom model is configured as a component with the name "my-realsense".
+- Ensure that the module executable is saved to a location on the filesystem of your robot that `viam-server` can access.
+  For example, if you are running `viam-server` on an Raspberry Pi, you must save the module executable on the Pi's filesystem.
+- Ensure that this file is executable (runnable) with the following command:
 
-``` shell
-realpath <path-to-your-module-directory>/<your-module>
-```
+   ``` shell
+   sudo chmod a+rx <path-to-your-module-executable>
+   ```
+
+See [Compile your module into an executable](/extend/modular-resources/create/#compile-the-module-into-an-executable) for more information.
+
+### Add a local module
+
+To add a local module on your robot:
+
+1. Navigate to the **Config** tab of your robot's page on [the Viam app](https://app.viam.com).
+   - If you are adding a modular [component](/components/), click the **Components** subtab and click **Create component**.
+   - If you are adding a modular [service](/services/), click the **Services** subtab and click **Create service**.
+
+1. Then, select the `local modular resource` type from the list.
+
+   {{<imgproc src="extend/modular-resources/configure/add-local-module-list.png" resize="300x" declaredimensions=true alt="The add a component modal showing the list of components to add with 'local modular resource' shown at the bottom">}}
+
+1. On the next screen:
+   - Select the type of modular resource provided by your module, such as a [camera](/components/camera/), from the drop down menu.
+   - Enter the {{< glossary_tooltip term_id="model-namespace-triplet" text="model namespace triplet">}} of your modular resource's [model](/extend/modular-resources/key-concepts/#models).
+     If you are adding a pre-built modular resource, the model triplet should be provided for you in the module's documentation.
+   - Enter a name for this instance of your modular resource.
+     This name must be different from the module name.
+
+   {{<imgproc src="extend/modular-resources/configure/add-local-module-create.png" resize="400x" declaredimensions=true alt="The add a component modal showing the create a module step for an intel realsense module">}}
+
+1. Click **Create** to create the modular resource provided by the local module.
 
-### Configure your module
+You can also add the module directly, without first adding its modular component or service:
 
-To configure your new *module* on your robot, navigate to the **Config** tab of your robot's page on [the Viam app](https://app.viam.com) and click on the **Modules** subtab.
+1. Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+1. Click on the **Modules** subtab.
+1. Scroll to the **Add local module** section.
+1. Enter a **Name** for this instance of your modular resource.
+1. Enter the [module's executable path](/extend/modular-resources/create/#compile-the-module-into-an-executable).
```

Comment:
Line 252-- What filesystem? Would say "your robot/machine's computer" or something like that (assuming that's correct if local module, if not needs other clarification)